src/routes/Player/VideosMenu/VideosMenu.js

```diff
@@ -0,0 +1,51 @@
+// Copyright (C) 2017-2022 Smart code 203358507
+
+const React = require('react');
+const PropTypes = require('prop-types');
+const classnames = require('classnames');
+const Video = require('../../MetaDetails/VideosList/Video');
+const styles = require('./styles');
+
+const VideosMenu = ({ className, metaItem, seriesInfo }) => {
+    const onMouseDown = React.useCallback((event) => {
+        event.nativeEvent.videosMenuClosePrevented = true;
+    }, []);
+    const videos = React.useMemo(() => {
+        return seriesInfo && seriesInfo.season && Array.isArray(metaItem.videos) ?
```

Comment:
check for `typeof seriesInfo.season === 'number'`
othersize special episodes will list all episodes